matter_server/server/device_controller.py

```diff
@@ -1116,7 +1118,14 @@ def resubscription_succeeded(
                 autoResubscribe=True,
             )
 
-        sub.SetAttributeUpdateCallback(attribute_updated_callback)
+        if not isinstance(sub, Attribute.SubscriptionTransaction):
+            # Aborted setups result in ReadResult instead of SubscriptionTransaction
+            # Probably a bug: https://github.com/project-chip/connectedhomeip/issues/33570
+            node_logger.warning("Subscription setup failed.")
+            return
+
+        sub.SetAttributeUpdateCallback(None)
```

Comment:
this feels a bit weird. Why is the callback explicitly set to None here ? Isn't it None/undefined by default ?

Reply:
Exactly, there is a default handler configured currently, which prints changes to stdout :see_no_evil: 

I could have patched that out in our CHIP SDK changes, but I wanted to keep changes minimal for now.

Reply:
can you leave a comment above this line then as a reminder for later ?